eip-progpow.md

```diff
@@ -0,0 +1,115 @@
+---
```

Comment:
Please format as per EIP-X and EIP-1: Headers should be in lowercase, and empty headers should be omitted.

Reply:
Thanks. Fixed.